packages/cli/src/lib/utils.test.ts

```diff
@@ -73,25 +75,33 @@ describe('utils', () => {
   });
 
   describe('getAxeSource', () => {
-    describe('mock file', () => {
-      beforeEach(() => {
-        mock({
-          '/node_modules/axe-core': {},
-          '../node_modules/axe-core': {
-            'axe.js': mock.load(require.resolve('axe-core'))
-          }
-        });
+    describe.only('mock file', () => {
```

Comment:
Stray `.only`

Reply:
(the `axe-core` repo uses `eslint-plugin-mocha-no-only` to prevent this class of mistake from being merged accidentally, maybe would be good for a separate PR to add that in this repo as well)